app/scenes/right_menu_drawer/right_menu_drawer_item.js

```diff
@@ -23,18 +23,23 @@ export default class MainMenuItem extends React.Component {
         style: React.PropTypes.oneOfType([
             React.PropTypes.object,
             React.PropTypes.array
-        ])
+        ]),
+        shouldRender: React.PropTypes.bool
     }
 
     render() {
+        const {onPress, style, children, shouldRender = true} = this.props;
```

Comment:
Can you use `defaultProps` for setting the default value of `shouldRender`? I didn't actually know you could set default values when destructuring.